### PR TITLE
[lipstick] Add support for querying the LauncherItems able to handle a MIME type. JB#53843 OMP#JOLLA-42

### PR DESCRIPTION
**Pull request by adenexter**
And also support for then starting those applications with arguments.

### CODE REVIEW

**Review by Tomin1 — APPROVED** [review]

File: src/src.pro [hunk 1]
```diff
@@ -163,6 +163,9 @@ packagesExist(contentaction5) {
     PKGCONFIG += contentaction5
     DEFINES += HAVE_CONTENTACTION
 } else {
+    PKGCONFIG += \
+        gio-2.0 \
+        gio-unix-2.0
```
Comment (okodron) [inline src/src.pro:168]: Better to add these dependencies to spec as well.

Comment (adenexter) [reply src/src.pro:168]: This the fallback for builds without libcontentaction, which is a dependency defined in the spec so this branch won't be evaluated.

I don't know why it's an optional dependency.

Comment (okodron) [reply src/src.pro:168]: Ok, got it.

**Review by okodron — APPROVED** [review]

File: src/src.pro (hunk 1 above)
Comment (Thaodan) [reply src/src.pro:168]: You could add a build condition to build with either of them.
See:
http://rpm.org/user_doc/conditional_builds.html

**Comment by okodron** [comment]
Previous discussion was here: https://git.sailfishos.org/mer-core/lipstick/merge_requests/180

**Pull request merged**